.gitignore

```diff
@@ -3,3 +3,5 @@
 .project
 *.DS_Store
 /vendor
+/tests/build
```

Comment:
I don't think these should be necessary. Any files that the tests generate _should_ be remove when the tests complete, via `cleanUp`.